src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -27,22 +27,21 @@ public final class SubmitYourAppealToCcdCaseDataDeserializer {
     private static final String YES = "Yes";
     private static final String NO = "No";
 
-
     private SubmitYourAppealToCcdCaseDataDeserializer() {
-        // Empty
     }
 
-    public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper, String region, RegionalProcessingCenter rpc,
-            boolean workAllocationEnabled) {
-        SscsCaseData caseData = convertSyaToCcdCaseData(syaCaseWrapper, workAllocationEnabled);
-
-        return caseData.toBuilder()
-                .region(region)
-                .regionalProcessingCenter(rpc)
-                .build();
+    public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper,
+                                                       String region,
+                                                       RegionalProcessingCenter rpc,
+                                                       boolean caseAccessManagementEnabled) {
+        return convertSyaToCcdCaseData(syaCaseWrapper, caseAccessManagementEnabled)
+            .toBuilder()
+            .region(region)
+            .regionalProcessingCenter(rpc)
+            .build();
     }
 
-    public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper, boolean workAllocationEnabled) {
+    public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper, boolean caseAccessManagementEnabled) {
```

Comment:
Method has become too bloated, consider refactor for easier readability

Reply:
Emm no.. not my method, that would take forever, unless we rescope this story?

Reply:
No, you're right. Leave it :)